tests/scripts/common_functions.sh

```diff
@@ -36,10 +36,20 @@ TESTS_FAILED=0
 TESTS_PASSED=0
 TESTS=0
 
-TEST_SUITE_ARRAY=("app_autotune_yaml_tests" "autotune_config_yaml_tests" "basic_api_tests")
+
+declare -A layer_configs=([petclinic-deployment-0]="container" [petclinic-deployment-1]="container" [petclinic-deployment-2]="container")
+deployments=("petclinic-deployment-0" "petclinic-deployment-1" "petclinic-deployment-2" "acmeair-deployment-0" "acmeair-deployment-1" "acmeair-deployment-2")
```

Comment:
Are these apps being deployed in this test? I don't see it.

Reply:
Yes ., Not need for this test . Removed